src/tables.jl

```diff
@@ -38,10 +38,11 @@ function Tables.columns(A::Union{KeyedArray, NdaKa})
     L = hasnames(A) ? (dimnames(A)..., :value) :
         (ntuple(d -> Symbol(:dim_,d), ndims(A))..., :value)
     R = keys_or_axes(A)
-    G = ntuple(ndims(A)) do d
-        vec([rs[d] for rs in Iterators.product(R...)])
-        # _vec(rs[d] for rs in Iterators.product(R...))
-    end
+    R_inds = map(eachindex, R)
```

Comment:
Sorry, can you clarify where the type instability was coming from and how this avoid it? Is just that the `ntuple(f, ndims(...))` call produces type instabilities that calling `map` avoids? If that's the case this feels like a hack around limitations with the language?

Reply:
It's precisely this, because `map` seems to be able to unroll. I've added a comment -- could you let me know if it clears this up at all?

Reply:
Awesome, thanks!